src/server/services/wms/qgswmsrendercontext.h

```diff
@@ -187,6 +193,12 @@ namespace QgsWms
       QgsAccessControl *accessControl() const;
 #endif
 
+      /**
+       * Returns a map having layer group names as keys and a list of layer instances as values.
```

Comment:
Stupid question, sorry, what's the difference between "layer instances" and "layers"?

Reply:
No difference, my brain was in OOP mode.